packages/compass-e2e-tests/positional-args/positional-args.go

```diff
@@ -0,0 +1,43 @@
+/*
+all this does it the shell one-liner:
+
+$BINARY "$@" $POSITONAL_ARGS
+
+Why? Because we have to get chromedriver to open chrome (actually electron) like
+that and shell scripts don't just work on windows.
+
+Why not just pass POSITIONAL_ARGS directly? Because something in the webdriverio
+-> webdriver -> chromedriver -> chrome (electron) chain is automatically
+prepending -- to all args that don't already have it and compass supports
+optional positional args that we're trying to write e2e tests for.
+*/
+
+package main
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+func main() {
+	binary := os.Getenv("BINARY")
+	positionalArgs := strings.Split(os.Getenv("POSITIONAL_ARGS"), "_pathsep_")
```

Comment:
Either go or windows didn't not play well with the fact that the env var was actually a bunch of space-separated vars - it sent that as one single command-line variable. Easiest way I could think of to still keep it just one env var is to join it together and split it.

Alternatively we can write it to a file as json  or encode it into an env var as json and then parse that json gain on the go side.. felt like overkill.